CSUB-19: More tests for min fees

atodorov opened the pull request.

atodorov committed:
tests: Use credal-js from git to avoid code duplication

b/c integration tests require multiple helper functions to create
required transactions before runing tests.

0xjeremyfrank reviewed (commented):
I'm not sure this is a good idea yet. I may change how things are done often as I try to figure out best approaches and I don't want it to break CI in this repos.

atodorov committed:
Rework tests to use credal-js

and make more use of test-utils functions too

atodorov committed:
Verify that AddDealOrder fee is min 0.01 CTC

atodorov committed:
Allow specifying of blockchain target URL

when updating Creditcoin type definitions. For example
https://testnet.creditcoin.network:9933

atodorov committed:
Verify that RegisterDealOrder fee is min 0.01 CTC

0xjeremyfrank reviewed (approved):
lgtm
